src/diagonal.works/b6/world.go

```diff
@@ -77,6 +77,23 @@ func (t *Tag) FromString(s string) {
 	t.Value = String(value)
 }
 
+func (t Tag) Equal(other Tag) bool {
+	if t.Key != other.Key {
+		return false
+	}
+	switch v := t.Value.(type) {
```

Comment:
y not just t.value == other.value

Reply:
It would work now, but if Value ends up implementing the Value interface by being a pointer to a struct, then == will just compare the pointer addresses, which is not what we want (or worse, panic if the underlying types aren't comparable), so I'm just preparing for the future given I needed to touch the code anyway.